bedrock/security/templates/security/bug-bounty/faq-webapp.html

```diff
@@ -150,10 +260,16 @@ <h1 itemprop="name" class="title-shadow-box">Web Application Security Bug Bounty
           listed?</dt>
         <dd class="answer">
           <p>If you find an issue with a site which is not “officially” part under the
-            web application bug bounty, we would still like to know. If the bug is
+            website and services bug bounty program, we would still like to know. If the bug is
             extraordinary, we might still consider the bug to be nominated for a bounty. In
             the past we have paid for interesting bugs which are outside of normal
-            policy.</p>
+            policy, but it's the rare exception and not the rule. It's hard to define 'extraordinary,' 
```

Comment:
Small nit - move comma outside of quotes, and use double quotes to be consistent with `"extraordinary"` below.